ci/unit_tests/test_endoscopic_tool_segmentation_dist.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+import torch
+from monai.data import PILWriter
+from parameterized import parameterized
+from utils import export_config_and_run_mgpu_cmd
+
+TEST_CASE_1 = [
+    {
+        "bundle_root": "models/endoscopic_tool_segmentation",
+        "train#trainer#max_epochs": 1,
+        "train#dataloader#num_workers": 1,
+        "validate#dataloader#num_workers": 1,
+        "train#deterministic_transforms#3#spatial_size": [32, 32],
+    }
+]
+
+TEST_CASE_2 = [
```

Comment:
here forgot to disable the load checkpoint handler, thus an error in:
https://github.com/Project-MONAI/model-zoo/actions/runs/6021868190/job/16335497561#step:2:875
Or I suggest to use test_order thing, it's helpful to actually load the model weights (can detect potential error)

Reply:
Saw your new commits, will resolve this conversation